Realm/RLMMongoCollection.h

```diff
@@ -18,12 +18,27 @@
 
 #import <Foundation/Foundation.h>
 #import <Realm/RLMObjectId.h>
+#import <Realm/RLMNetworkTransport.h>
```

Comment:
Any way we can avoid this import here?

Reply:
No because we need `RLMEventDelegate`

Reply:
Can't we forward declare it?

Reply:
Forward declaring it gives a build error basically stating the delegate type cannot be found. Same goes for trying to forward declare the protocol.

Reply:
Declaring a conformance to a protocol is one of the things that needs the full definition.